### ISSUE TITLE
please update mod

### ISSUE THREAD

**Issue by kaizuupe** [Mydayyy/Valheim-ServerSideMap#69]
Hi hello, any info about update this mod to the current version?
would love to have it.
Since after the update, we got FPS Drops and make it unplayable using the mod.

got this error when try to use our own world (local)
﻿[Error  : Unity Log] MissingMethodException: Method not found: void .ZNet.Save(bool,bool)
Stack trace:
ServerSideMap.SaveWorld+ZnetPatchLoadMap.Postfix (ZNet __instance) (at <6c6d7457ac254564a07af03a22a0254f>:0)
(wrapper dynamic-method) ZNet.DMD<ZNet::LoadWorld>(ZNet)
ZNet.ServerLoadWorld () (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
(wrapper dynamic-method) ZNet.DMD<ZNet::Start>(ZNet)

but when try to put it on the server host, we cant connect to the server

**Comment by Mydayyy** [comment]
Greetings,

thank you for the information. I am currently fairly busy but I will try looking into updating the mod on the weekend.

Best Regards
Mydayyy

**Comment by kaizuupe** [comment]
Thanks for replying, I beg you to fix it anytime you have time

**Comment by BradLugo** [comment]
I ran into a similar issue. Long story short, updating BepInEx to the latest version **without Thunderstore/r2modman** fixed the problem for me (I'm using [v5.4.23.2](https://github.com/BepInEx/BepInEx/releases/tag/v5.4.23.2))

I didn't do much diagnosing of the problem after I got this mod working, but my guess is that the version of BepInEx in the mod stores isn't _quite_ right (maybe there's something wonky with the v5.4.2202 patch in particular or maybe there was a bug that got resolved in a newer version of BepInEx? - not sure). Possible lead if someone wants to continue looking into this: https://github.com/BepInEx/BepInEx/issues/699

**Comment by Mydayyy** [comment]
Greetings,

I tested the mod today and it did work for me on the newest Valheim Update (0.219.14). I tested pin sharing as well as map exploration on a dedicated server. Can you confirm your client version and your server version? And would you be able to provide additional logs?

Best Regards
Mydayyy

**Comment by icemansparks** [comment]
Complete log (some other mods installed) loading into an old world which was not yet opened since bogwitch update:
[LogOutput.log](https://github.com/user-attachments/files/17691346/LogOutput.log)

There doesn't seem to be a general Error when loading the game, but loading a mao just runs into a dead loop without ever completing the loading screen.

Game and server are both up to date with latest stable patch branch.

**Comment by Kvnsky** [comment]
Getting the same issue on:
- Valheim 0.219.14
- ServerSideMap 1.3.11
- Clean install (no other mods)

Two different issues:
- In singleplayer: Getting the error when trying to load any world
- On server: Server prints the error once while starting, then clients can't connect at all

Full error (appears in both scenarios):
```
[Error  : Unity Log] MissingMethodException: Method not found: void .ZNet.Save(bool,bool)
Stack trace:
ServerSideMap.SaveWorld+ZnetPatchLoadMap.Postfix (ZNet __instance) (at <6c6d7457ac254564a07af03a22a0254f>:0)
(wrapper dynamic-method) ZNet.DMD<ZNet::LoadWorld>(ZNet)
ZNet.ServerLoadWorld () (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
ZNet.Start () (at <94a8ab08dca748d3beb540d5228ac7b0>:0)
```

**Comment by Mydayyy** [comment]
Greetings,

I am currently still having trouble reproducing this. I will continue to try more things.

Best Regards
Mydayyy

**Comment by kaizuupe** [comment]
Thanks for giving it a shot. Hopefully, you can figure out another way to fix this. We're all rooting for your mod! It's awesome and makes exploring so much fun.

**Comment by Mydayyy** [comment]
Greetings,

would it be possible to test this debug build? I changed the signature of the .save method. In case this works I am still clueless as to why the error did not happen for me but both builds are working for me right now.

[ServerSideMap.zip](https://github.com/user-attachments/files/17777826/ServerSideMap.zip)

**Comment by Flashminat0** [comment]
Hi, 

I had the same issue as OP and this last .dll file seems to fix the issue and I've tested with one of my friends on our server.
Thanks a lot, Much appreciated <3

**Comment by Mydayyy** [comment]
Greetings,

I released the update as v1.3.12 here, on NexusMods and on Thunderstore.

Best Regards
Mydayyy
